src/core/deps/DependencyGraph.ts

```
@@ -0,0 +1,206 @@
+    for (const n of nodes) out.set(n, new Set());
+    for (const [from, tos] of this.adj.entries()) {
+      for (const to of tos) {
+        if (!out.has(to)) out.set(to, new Set());
+        out.get(to)!.add(from);
+      }
+    }
+
+    // Compute indegree in reversed graph
+    const indeg: Map<string, number> = new Map();
+    for (const n of nodes) indeg.set(n, 0);
+    for (const [u, neigh] of out.entries()) {
```

ESLint

[error] 'u' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)